fronts-client/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -227,176 +224,156 @@ class Collection extends React.Component<CollectionProps, CollectionState> {
 		const isUneditable = isCollectionLocked || browsingStage !== cardSets.draft;
 
 		return (
-			<CollectionDisplay
-				frontId={frontId}
-				id={id}
-				browsingStage={browsingStage}
-				isUneditable={isUneditable}
-				isLocked={isCollectionLocked}
-				isOpen={isOpen}
-				hasMultipleFrontsOpen={hasMultipleFrontsOpen}
-				onChangeOpenState={() => onChangeOpenState(id, isOpen)}
-				headlineContent={
-					hasUnpublishedChanges &&
-					canPublish && (
-						<Fragment>
-							<EditModeVisibility visibleMode="editions">
-								{!isFeast && (
-									<HeadlineContentButton
-										priority="default"
-										onClick={() => this.props.setHidden(id, !isHidden)}
-										title="Toggle the visibility of this container in this issue."
-									>
-										{isHidden ? 'Unhide' : 'Hide'}
-									</HeadlineContentButton>
-								)}
-								{isFeast && (
-									<>
-										<MoveButtonsContainer>
-											<ButtonCircularCaret
-												small
-												openDir="up"
-												disabled={!this.props.canMoveUp}
-												onClick={() =>
-													this.props.moveFrontCollection(
-														this.props.frontId,
-														this.props.id,
-														'up',
-													)
-												}
-											/>
-											<ButtonCircularCaret
-												small
-												disabled={!this.props.canMoveDown}
-												onClick={() =>
-													this.props.moveFrontCollection(
-														this.props.frontId,
-														this.props.id,
-														'down',
-													)
-												}
-											/>
-										</MoveButtonsContainer>
+			<>
+				<CollectionDisplay
+					frontId={frontId}
+					id={id}
+					browsingStage={browsingStage}
+					isUneditable={isUneditable}
+					isLocked={isCollectionLocked}
+					isOpen={isOpen}
+					hasMultipleFrontsOpen={hasMultipleFrontsOpen}
+					onChangeOpenState={() => onChangeOpenState(id, isOpen)}
+					headlineContent={
+						hasUnpublishedChanges &&
+						canPublish && (
+							<Fragment>
+								<EditModeVisibility visibleMode="editions">
+									{!isFeast && (
+										<HeadlineContentButton
+											priority="default"
+											onClick={() => this.props.setHidden(id, !isHidden)}
+											title="Toggle the visibility of this container in this issue."
+										>
+											{isHidden ? 'Unhide' : 'Hide'}
+										</HeadlineContentButton>
+									)}
+									{isFeast && (
+										<>
+											{targetedRegions?.length > 0 ? 'US!' : ''}{' '}
```

Comment:
let's replace this text with a US flag emoji: https://emojipedia.org/flag-united-states

Reply:
bonus if you can get a tooltip on there to say "Targeted to US only" :grin: